### ISSUE TITLE
Graph Zoom Range is not updated

### ISSUE THREAD

**Issue by eschoeller** [Cacti/cacti#3029]
In the dark theme, when selecting a graph range using the manual zoom selection feature, the first attempt the graph is not updated. Only the second (and subsequent) attempts actually update the graph. 
See this example:

![Peek 2019-10-13 12-13](https://user-images.githubusercontent.com/12864741/66719898-f4e32800-edb2-11e9-891f-315eb3132428.gif)

And then the next attempt (had to split it since Peek couldn't handle the length of the clip):
![Peek 2019-10-13 12-14](https://user-images.githubusercontent.com/12864741/66719901-0f1d0600-edb3-11e9-8628-703af3a0caf6.gif)

**Comment by cigamit** [comment]
Check for a Javascript error in the console when reproducing that.

**Comment by eschoeller** [comment]
```
jquery.js?2f0d15c70b7e652d5d61ed4032eb44af:9524 [Violation] 'load' handler took 202ms
jquery.js?2f0d15c70b7e652d5d61ed4032eb44af:3600 [Violation] 'setTimeout' handler took 1089ms
[Violation] Forced reflow while executing JavaScript took 531ms
```

**Comment by netniV** [comment]
that sounds like one of those new browser features that was introduced to stop them being unresponsive due to excessive JavaScript CPU usage.  The real question is whether that is down to it waiting for a response via ajax from Cacti or just processing the data that's returned.

**Comment by cigamit** [comment]
None of those are fatal.  We should look at the response from the network tab and see if it's getting good json.

**Comment by netniV** [comment]
Yes, that is a good point as we have found that PHP likes to output warnings in the middle of the output.

**Comment by cigamit** [comment]
Those warnings are generated right inside of Chrome stating that Chrome has detected that the script took too much time.  They are not PHP.

So, @eschoeller, what needs to come next is to open the network debug panel before the first zoom, and then perform the zoom, open the callback params and the json response, and post the screen shots.

**Comment by eschoeller** [comment]
![image](https://user-images.githubusercontent.com/12864741/67148710-bc829480-f25f-11e9-8caf-5d3fbecf2705.png)
So ... where to next? I don't work with the developer console too often.

**Comment by netniV** [comment]
> Yes, that is a good point as we have found that PHP likes to output warnings in the middle of the output.

Yeah ignore that, that response was supposed to be for invalid JSON... somehow I think I posted it to the wrong thread.

**Comment by netniV** [comment]
the tine column is obscured because it was not wide enough, can you adjust the columns on the next screen shot so that it clearly shows how long each request has taken @eschoeller ?

**Comment by eschoeller** [comment]
![image](https://user-images.githubusercontent.com/12864741/67166345-8de1e800-f34b-11e9-9b7a-30526f3baaaa.png)
![image](https://user-images.githubusercontent.com/12864741/67166354-a18d4e80-f34b-11e9-8ab7-f422cc15c249.png)

**Comment by eschoeller** [comment]
That's captured from an instance where the zoom worked. This is when it didn't:
![image](https://user-images.githubusercontent.com/12864741/67166371-df8a7280-f34b-11e9-877a-22a07ae559e7.png)
![image](https://user-images.githubusercontent.com/12864741/67166383-f4670600-f34b-11e9-9602-0f0608b2630f.png)

**Comment by cigamit** [comment]
@eschoeller, we will have to do a WebEx or something on this one.

**Comment by eschoeller** [comment]
OK, sounds good.

**Comment by cigamit** [comment]
@eschoeller, test with the latest jquery.zoom.js, I think that this was resolved in #3112.  Close if that's the case.

**Comment by cigamit** [comment]
@eschoeller, looking for an update.  Hope all is well.

**Comment by eschoeller** [comment]
I still see a lot of inconsistencies with how the dark theme handles graph zooming. I need to update to a recent code version though. Your call if you need to close this and then I re-open later and reference this issue.

**Comment by cigamit** [comment]
Pretty sure it's fixed @eschoeller.  We found an issue where calls we being interrupted and fixed it in the 1.2.29 code base.
